compliance/store/src/test/java/org/eclipse/rdf4j/repository/sail/nativerdf/ForwardChainingSchemaCachingRDFSInferencerNativeRepositoryConnectionTest.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.repository.sail.nativerdf;
+
+import org.eclipse.rdf4j.IsolationLevel;
+import org.eclipse.rdf4j.common.io.FileUtil;
+import org.eclipse.rdf4j.repository.RDFSchemaRepositoryConnectionTest;
+import org.eclipse.rdf4j.repository.Repository;
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+import org.eclipse.rdf4j.sail.inferencer.fc.ForwardChainingSchemaCachingRDFSInferencer;
+import org.eclipse.rdf4j.sail.nativerdf.NativeStore;
+
+import java.io.File;
+import java.io.IOException;
+
+public class ForwardChainingSchemaCachingRDFSInferencerNativeRepositoryConnectionTest extends RDFSchemaRepositoryConnectionTest {
+
+	private File dataDir;
```

Comment:
Nitpick, this is better handled using the JUnit TemporaryFolder rule in the long term but it isn't the only use of this pattern so still okay. An example of one that has been migrated:

https://github.com/eclipse/rdf4j/blob/a283e758fb79e7b62e94b669547e6e64c64106cf/compliance/store/src/test/java/org/eclipse/rdf4j/sail/nativerdf/NativeStoreConcurrencyTest.java#L28